src/alerting/components/AlertHistoryControls.tsx

```diff
@@ -9,7 +9,7 @@ import {Page} from '@influxdata/clockface'
 
 // Types
 import {AlertHistoryType} from 'src/types'
-import {EventViewerChildProps} from 'src/eventViewer/types'
+import {EventViewerChildProps} from 'src/types/eventViewer'
```

Comment:
nit: updating `src/types/index` to export the `eventViewer` inside the `index` file so that we can import this directly from `src/types` to stay consistent with the rest of the app

Reply:
not a nit-pick. definitely useful, helpful and knowledge
🙇‍♂️